common/frame.go

```diff
@@ -1030,12 +1028,12 @@ func (f *Frame) innerHTML(selector string, opts *FrameInnerHTMLOptions) (string,
 	if v == nil {
 		return "", nil
 	}
-	gv, ok := v.(goja.Value)
+	gv, ok := v.(string)
 	if !ok {
-		return "", fmt.Errorf("getting inner html of %q: unexpected type %T", selector, v)
+		return "", fmt.Errorf("unexpected type %T", v)
```

Comment:
This error was stuttering "getting inner html.." in its caller. So, I've removed it from here. There are similar cases in the rest of the code and I've fixed them as well.